### ISSUE TITLE
Output of field instructions

### ISSUE THREAD

**Issue by mark-adigital** [roundhouse/FormBuilder-2-Craft-CMS#148]
Think I've just spotted a small issue - the default input templates appear to be outputting the instruction text in such a format that the surrounding HTML tags (`<p>` etc) are displayed as plaintext. You can see examples here: https://www.mint-homes.co.uk/property/walney-barrow/tenancy-application

Don't recall seeing this previously, so guessing it has been introduced with some relatively recent update.

Thanks

**Comment by EtienneBerlin** [comment]
I am running into the same issue. My previously styled fieldnames/descriptions are now showing unstyled with html-tags in plain text displayed in the frontend.

**Comment by kingkelly** [comment]
I updated FormBuilder and am also getting the <p> tags visible for the instructions.

Is there a fix for this? Ive got a client complaining about this, and not sure where to made the edit.

**Comment by mark-adigital** [comment]
I don't think this was ever going to get fixed, so I created a [pull request](https://github.com/roundhouse/FormBuilder-2-Craft-CMS/pull/155) for it.
